xlive/Blam/Engine/interface/screens/screen_main_menu.cpp

```diff
@@ -509,7 +509,7 @@ static bool __cdecl screen_show_screen_4way_signin_xbox_live_callback(void)
 			_user_interface_channel_type_game_error,
 			_ui_error_xblive_user_not_authorized,
 			_window_4,
-			NONE,// allow all controllers
+			(uint16)NONE,// allow all controllers
```

Comment:
?

Reply:
this was throwing a warning that prevented compilation at one point, I am not sure why it's not throwing the warning anymore will revert.